autouri/gcsuri.py

```diff
@@ -166,6 +166,22 @@ def read(self, byte=False):
             return b
         return b.decode()
 
+    def find_all_files(self):
+        cl = GCSURI.get_gcs_client(self._thread_id)
+        bucket, path = self.get_bucket_path()
+        sep = GCSURI.get_path_sep()
+        if path != '':
+            path = path.rstrip(sep) + sep
+
+        result = []
+        blobs = cl.list_blobs(bucket, prefix=path)
+        if blobs:
+            for blob in blobs:
+                scheme = GCSURI.get_schemes()[0]
```

Comment:
This is a little funny to me. But out of the scope of this PR probably. It seems like you'd want to be able to use `self.SCHEME` or `cls.SCHEME` or something.